bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java

```diff
@@ -318,51 +320,39 @@ public int hashCode() {
 
     }
 
-    private final Collection<ChannelGroupType> channelGroupTypes;
-    private final Collection<ChannelType> channelTypes;
+    private static final Collection<ChannelGroupType> CHANNEL_GROUP_TYPES = Collections.emptyList();
+    private static final Collection<ChannelType> CHANNEL_TYPES = Collections
```

Comment:
```
Arrays.asList(SYSTEM_CHANNEL_SIGNAL_STRENGTH, SYSTEM_CHANNEL_LOW_BATTERY, ...);
```

would be possible to.

Reply:
But then it is not an unmodifieable list anymore... Do you want me to change to anyway, or leave it?

Reply:
No need to change anything.

I just thought about replacing
```Java
Stream.of(a,b,c,d).collect(Collectors.toList());
```
with
```Java
Arrays.asList(a,b,c,d);
```
because it is shorter.

The wrapping unmodifiable view creation could stay.